.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### Description
+
+<!--
+Please describe your pull request and what relevant information it adds to the documentation.
+
+Make sure to read the contributing guide:
+https://github.com/discord/discord-api-docs/blob/master/CONTRIBUTING.md
+
+If you believe that your pull request fulfills these requirements, please submit it! We <3 corrections and improvements!
+-->
+
+### Checklist
+
+- [ ] This PR fixes inaccurate/outdated information from the documentation.
+- [ ] This PR adds clarification to complicated/unclear explanations.
+- [ ] This PR adds more helpful context to incomplete documentation.
+- [ ] This PR fixes spelling and grammatical errors.
+- [ ] These changes reflect the current default version of the API.
```

Comment:
Please do not make these a checklist, checklists show up very poorly if you're not using them as checklists.

I think this section should be called status, the suggestions should be moved into a comment so that people can move out relevant ones and not display irrelevant ones

Furthermore, the changes reflecting the current default version of the API is probably not a good suggestion. The default version is still v6 (deprecated) which is not what the documentation currently reflects.

Reply:
> the changes reflecting the current default version of the API

it should probably say "the latest version" instead

Reply:
Checklists are not suited to things that aren't meant to be completed, not every one of these applies to every PR (in fact only 2 will usually apply)

For example, none of them should be checked in this PR

Reply:
Yeah, I'm in agreement with ckohen here - checklists are great for things like "I've read the contributing guide" and "My pull request has a clear and specific title", but not for options like you've added here.

These would be acceptable radio buttons, but GH doesn't support that (yet.) so I'd argue to remove these.